toc/working-groups/foundational-infrastructure.md

```diff
@@ -204,7 +204,6 @@ areas:
   - cloudfoundry/stembuild
   - cloudfoundry/stemcells-alicloud-index
   - cloudfoundry/socks5-proxy
-  - cloudfoundry/system-metrics-server-release
```

Comment:
This should probably be changed to: https://github.com/cloudfoundry/bosh-system-metrics-server-release